torch/_inductor/kernel/flex/flex_flash_attention.py

```diff
@@ -211,6 +267,23 @@ def create_flex_flash_attention_kernel(
         NEEDS_BLOCK_MASK=needs_block_mask,
     )
 
+    def wrap_choice_render(choice):
+        original_make_kernel_render = choice.make_kernel_render
```

Comment:
Typing nit. Extract ParamSpec using typing callable asssigment on this line and use it to forward args in the function below.

Using :Callable[ParamSpec, R] = to extract the type info.